tlsfuzzer/expect.py

```diff
@@ -515,6 +515,10 @@ class ExpectServerHello(_ExpectExtensionsMessage):
     """
     Parsing TLS Handshake protocol Server Hello messages.
 
+    Processing of the ServerHello message updates the record layer
+    to the version advertisied by server.
+    Use :py:class:`~tlsfuzzer.messages.SetRecordVersion` to change it earlier.
```

Comment:
This "earlier" gives me pause, does it mean that if you use SetRecordVersion the processing of ServerHello will not change the version ?

Reply:
no, it will still update it, this is for cases when you want to send multiple records with different versions before processing ServerHello

Reply:
Should probably be explained with these many words then.